index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Trirong">
+    <title>News-site-week-two</title>
+    <style>
+        body {
+          font-family: "Trirong", serif;
+        }
+    </style>
```

Comment:
They say that styling is better to be in css file, so you could move this to css like this:

body {
           font-family: "Trirong", serif;
         }